### DISCUSSION TITLE
Let's shutdown gitter

### DISCUSSION THREAD

**Discussion by vuolter** [pyload/pyload#4484] [Choices]
Years ago, Gitter, thanks to the great integrations with github, was definitely a good platform to create and manage a community for a foss project like pyLoad (so we did), but today imo there are better alternatives (like discord) and, thus, our community on gitter looks pretty "dead" these days...

Maybe we should focus just on discussions here on github.
What do you think people?

@GammaC0de

**Comment by GammaC0de** [comment]
> Maybe we should focus just on discussions here on GitHub.

Does discussions supports private discussions / chats?

**Comment by mihawk90** [comment]
I didn't even know the Gitter was still alive because for as long as I've been using pyload, the link in the Info (which is also in the issue template) was dead:

https://github.com/pyload/pyload/blob/4159a1191ec4fe6d927e57a9c4bb8f54e16c381d/.github/ISSUE_TEMPLATE/config.yml#L4

https://github.com/pyload/pyload/blob/4159a1191ec4fe6d927e57a9c4bb8f54e16c381d/src/pyload/webui/app/themes/pyplex/templates/info.html#L31

https://github.com/pyload/pyload/blob/4159a1191ec4fe6d927e57a9c4bb8f54e16c381d/src/pyload/webui/app/themes/modern/templates/info.html#L31

https://github.com/pyload/pyload/blob/4159a1191ec4fe6d927e57a9c4bb8f54e16c381d/src/pyload/webui/app/templates/info.html#L26

So I'm not sure how to even get there currently...

**Comment by vuolter** [comment]
@GammaC0de as I know only public chats are supported right now.
@mihawk90 I think there's a typo, the working link is https://gitter.im/pyload/pyload.

**Comment by vuolter** [comment]
Old gitter rooms have been archived.
